src/main/java/de/mediathekview/mserver/crawler/mdr/tasks/MdrTopicPageTask.java

```diff
@@ -0,0 +1,45 @@
+package de.mediathekview.mserver.crawler.mdr.tasks;
+
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractRecrusivConverterTask;
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import de.mediathekview.mserver.crawler.mdr.MdrConstants;
+import de.mediathekview.mserver.crawler.mdr.parser.MdrTopic;
+import de.mediathekview.mserver.crawler.mdr.parser.MdrTopicPageDeserializer;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.jsoup.nodes.Document;
+
+public class MdrTopicPageTask extends AbstractDocumentTask<CrawlerUrlDTO, CrawlerUrlDTO> {
+
+  public MdrTopicPageTask(AbstractCrawler aCrawler,
+      ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDtos) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDtos" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)